src/components/navigation/AppNavigation.vue

```diff
@@ -23,6 +23,12 @@
 <template>
 	<div id="app-navigation" :class="{'icon-loading': loading}">
 		<ul id="deck-navigation">
+			<AppNavigationBoardCategory
+				id="deck-navigation-dashboard"
+				:text="t('deck', 'Card Dashboard')"
```

Comment:
How about 'Your cards'. I'd rather avoid the dashboard term in the user interface in regards to https://github.com/nextcloud/server/pull/21346